src/Particle.js

```diff
@@ -590,7 +590,9 @@ class Particle {
 	downloadFirmwareBinary({ binaryId, auth }){
 		const uri = `/v1/binaries/${binaryId}`;
 		const req = request('get', uri);
-		req.use(this.prefix);
+		if (this.prefix) {
+			req.use(this.prefix);
+		}
 		this.headers(req, auth);
```

Comment:
i was seeing failures related to this line as well - see [CH story](https://app.clubhouse.io/particle/story/45862/downloadproductfirmware-and-downloadfirmwarebinary-fail-w-typeerror-fn-is-not-a-function) - are you?